CHANGELOG.md

```diff
@@ -7,8 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Fixed
+
 - Fixed request crash with empty body and unexpected header Content-Type (#189)
 
+### Added
+
+- `roles.httpd` role to configure one or more HTTP servers (#196)
+- `httpd:delete(name)` method to delete named routes (#197)
```

Comment:
I have never seen anyone using `name` option for a route. Until now, I didn't know it even exists. As far as I understand, it actually works not only with named routes, but with unnamed ones as well by using path as a key?

Reply:
No, we can't use just path. Routes are stored as `opts` (which is a table of many fields, some of them are filled inside of `add_route` function).
https://github.com/tarantool/http/blob/2e3dbd2935b3ecf8ce5e4eae62933762e0ee576a/http/server.lua#L1248-L1259

And those `opts` stored in the map with the `name` as a key. If there are no `name`, then we can't get `opts`.

Reply:
Oh, I see that metrics-export-role sets the name as well
https://github.com/tarantool/metrics-export-role/blob/36130b73c0a5975ff26f0c26f78ee7a368c1207b/roles/metrics-export.lua#L265

Reply:
The only problem is to create a nice API for the call. Maybe:
```
http:delete(path, name)
http:delete(path)
http:delete(nil, name)
```

Reply:
> Yes, it is. It uses name, BUT we could delete a route by path too. I don't see any problems.

I'm not sure, but as far as I remember that httpd has that weird thing that allows to set multiple handlers on the same route since route is not a unique identificator (and only one of handlers will be invoked).